solutions/swb-reference/integration-tests/tests/multiStep/user.test.ts

```diff
@@ -26,83 +27,140 @@ describe('multiStep users integration test', () => {
+      await adminSession.resources.users.create({
+        ...mockUserInput,
+        email: `success+activate-deactivate-user-${uuidv4()}@simulator.amazonses.com`
+      });
+      const listResponse = await adminSession.resources.users.get({ pageSize: 1 });
+      expect(listResponse.data.users.data.length).toEqual(1);
+      const user1Id = listResponse.data.users.data[0].id;
+      const paginationToken = listResponse.data.users.paginationToken;
+      expect(paginationToken).toBeDefined();
+
+      const listResponse2 = await adminSession.resources.users.get({ pageSize: 1, paginationToken });
+      expect(listResponse2.data.users.data.length).toEqual(1);
+      const user2Id = listResponse2.data.users.data[0].id;
+      expect(user1Id).not.toEqual(user2Id);
+
+      console.log('ProjectAdmin getting a user by userId');
+      const { data: acutalUserGetByPA } = await pa1Session.resources.users.user(mockUserId).get();
+      expect(acutalUserGetByPA).toMatchObject(mockUser);
+
+      console.log('Researcher has cannot call get a user by userId');
+      try {
+        await rs1Session.resources.users.user(mockUserId).get();
+      } catch (e) {
+        new HttpError(403, {
+          error: 'User is not authorized'
+        });
+      }
+
+      console.log('ITAdmin can update the user');
+      const updateMockUserInput = { firstName: 'updatedFirstName', lastName: 'updatedLastName' };
+      await adminSession.resources.users.user(mockUserId).update(updateMockUserInput, true);
+      const { data: updatedUser } = await adminSession.resources.users.user(mockUserId).get();
+      expect(updatedUser).toMatchObject({
+        ...updateMockUserInput,
+        id: mockUserId
+      });
+
+      console.log('PA cannot update the user');
+      try {
+        await pa1Session.resources.users.user(mockUserId).update(updateMockUserInput, true);
+      } catch (e) {
+        checkHttpError(
+          e,
+          new HttpError(403, {
+            error: 'User is not authorized'
+          })
+        );
+      }
+
+      console.log('researcher cannot update the user');
+      try {
+        await rs1Session.resources.users.user(mockUserId).update(updateMockUserInput, true);
+      } catch (e) {
+        checkHttpError(
+          e,
+          new HttpError(403, {
+            error: 'User is not authorized'
+          })
+        );
+      }
+
+      console.log('PA cannot delete the user');
+      try {
+        await pa1Session.resources.users.user(mockUserId).purge();
+      } catch (e) {
+        checkHttpError(
+          e,
+          new HttpError(403, {
+            error: 'User is not authorized'
+          })
+        );
+      }
+
+      console.log('researcher cannot delete the user');
+      try {
+        await rs1Session.resources.users.user(mockUserId).purge();
+      } catch (e) {
+        checkHttpError(
+          e,
+          new HttpError(403, {
+            error: 'User is not authorized'
+          })
+        );
+      }
```

Comment:
These seem redundant with the negative tests being run in the isolated users tests.

Reply:
Actually I see that updateUser.test.ts was not updated with the negative test cases. I'd recommend moving those there and taking these out here.